_config.yml

```diff
@@ -124,7 +124,7 @@ footerBlocks:
  -
   title: "FAQ"
   links:
-   - {permalink: "/cod/", text: "Code of Conduct"}
```

Comment:
The original "cod" looks like a typo for "coc". Anyone know if it had a meaning?

Reply:
Not sure, but it didn't make sense to me, hence the 'e'.

Reply:
Not sure. We inherited it from the original repository, I think. It didn't make sense to me, hence the 'e'.

Reply:
I would add a redirect from the strange URL if this is to be merged.

Reply:
Good point. And to be clear -- I did specifically add the old CoC to the repository when I created the website. I wanted to strengthen the CoC in light of current events.